azurerm/internal/services/network/tests/vpn_server_configuration_resource_test.go

```diff
@@ -98,6 +98,25 @@ func TestAccAzureRMVPNServerConfiguration_radius(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVPNServerConfiguration_multipleRadius(t *testing.T) {
```

Comment:
Could we add some more tests?

Going from 1 server with the old block -> 1 server with the new block, updating the new block from 1 -> 2 -> 1 -> none?

Reply:
@katbyte  tests has been updated.